scaling_performance/optimizing_storage.adoc

```diff
@@ -13,6 +13,81 @@ toc::[]
 
 == Optimizing Storage
 
+== General Storage Guidance
+
+The following is a matrix which details the storage backend types {product-title} can be used with implementation examples and vendors.
+
+=== Available Storage Options
+[format="csv",width="70%",cols="3"]
+[options="header]
+|=======================
+Type,Examples,Vendors
+hostPath,"hostPath pod mounted volumes.  A specific type of Block Storage.  File paths on the node mounted onto running pods.",Others
+Block,"Block Storage. HDD, SSD, NVMe, Raw Physical Disk, VM Physical Disk, VMDK, LVM, iSCSI, Fibre Channel, Ceph RBD, OpenStack Cinder, Dell/EMC Scale.IO, AWS EBS ^(1)^, GCE Persistent Disk ^(1)^, Azure Disk","RedHat, Seagate, WesternDigital, Toshiba, HGST, Intel, Samsung, OCZ, Plextor, PNY, Kingston, SanDisk, AWS, GCE, VMware, Dell/EMC, HPE, Others"
+File,"Network Attached File Storage.  CNS GlusterFS ^(1)^, GlusterFS ^(1)^, RHEL NFS, NetApp NFS ^(2)^, Vendor NFS, Vendor GlusterFS ^(3)^, Loopback Image over NFS","RedHat, NetApp, AWS, GCE, Others"
+Object,"Object Storage.  Ceph Object Storage, OpenStack Swift,  AWS S3, Google Cloud Storage, Vendor S3 ^(3)^, Vendor Swift ^(3)^","RedHat, AWS, GCE, Others"
+|=======================
+NOTE: ^(1)^ Ceph RBD, OpenStack Cinder, AWS EBS, GCE Persistent Disk support Dynamic PV Provisioning natively in {product-title}.
+
+NOTE: ^(2)^ NetApp NFS supports Dynamic PV Provisioning when using the Trident Plugin
+
+WARNING: ^(3)^ Vendor GlusterFS, Vendor S3, and Vendor Swift supportability and configurability may vary.
+
+=== Storage Backend Recommendations
+Below is a matrix that summarizes which storage backend types are Recommended, Supported, and Configurable for the given
+{product-title} application and useful attributes.
+[format="csv",width="70%",cols="10"]
+[options="header]
+|=======================
+Type,Tied to Node?,ROM,RWX,Registry,Scaled Registry,etcd,Metrics,Logging,Apps
```

Comment:
I would remove the app column, because the recommendation is really app dependent